airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -177,6 +181,7 @@ def __init__(self,  # pylint: disable=too-many-arguments,too-many-locals
                  full_pod_spec: Optional[k8s.V1Pod] = None,
                  init_containers: Optional[List[k8s.V1Container]] = None,
                  log_events_on_failure: bool = False,
+                 log_container_statuses_on_failure: bool = False,
```

Comment:
wouldn't True be more useful a default (and for log events too)

Reply:
I do think these logs are useful info and absolutely no harm in having them on by default. But I can see why they defaulted to False not to contaminate logs unless the task has a problem and you will turn it on then.

Reply:
In my opinion having both `log_events_on_failure` and `log_container_statuses_on_failure` be True makes more sense to me since printing out a few extra logs is close to zero cost and doesn't add that much clutter. While it looks like it's set to False to be aligned with the precedent set in code, I'm in favor of switching both to True.